NickvisionCavalier.GNOME/Blueprints/drawing_view.blp

```diff
@@ -15,6 +15,19 @@ Gtk.Stack _root {
     };
   }
 
+  Gtk.StackPage _welcomePage {
+    name: "welcome";
+    child: Adw.Bin {
+      Adw.StatusPage {
```

Comment:
I'd wrap this in a `Gtk.ScrollWindow` because on first startup, my config had a smaller window size and I was only able to see a part of the emoji and wasn't able to scroll around...had to resize the window.

Reply:
I think you won't be able to scroll it anyway because there's an overlayed gtk.WindowHandle... Yeah, weird layout

Reply:
ah ok no worries then